### DISCUSSION TITLE
Podcast Support

### DISCUSSION THREAD

**Discussion by rbharathkumar** [strawberrymusicplayer/strawberry#1026] [Q&A]
Hi all,
Thank you so much for the efforts in making this wonderful music player.
I have a small question.
Does strawberry have podcast support ?
Would it be possible to add support if it is something that aligns with the project's goals and timelines ?

Currently not many music players have proper podcast support with streaming, queueing, import/export (opml), sleep and playback timer options. So was just wondering...

Anyways the player is awesome as it is. thank you once again.

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
